kolibri/plugins/setup_wizard/middleware.py

```diff
@@ -29,6 +29,9 @@ def process_request(self, request):
 
         # Don't redirect for URLs that are required for the setup wizard
         allowed_paths = [reverse(name) for name in ALLOWED_PATH_LIST]
+
+        # manually add session bootstrapping api call to allowed path list
+        allowed_paths.append(reverse('session-detail', kwargs={'pk': 'current'}))
```

Comment:
I need to pass in the `kwargs` to reverse in order to get the correct url. The rest of path lists don't need to use `kwargs` to get the reversed url, but for session we need to.